recipes/search_avail_poi_lists.py

```diff
@@ -0,0 +1,7 @@
+from locussdk import get_avail_poi_lists
+
+
+def search_avail_poi_lists(name):
+    global_lists = get_avail_poi_lists(list_type='global')
+
+    return global_lists[global_lists['name'].str.contains(name)]
```

Comment:
Including `case` parameter for `str.contains()` to add control for case sensitivity and broadening search scope 
i.e. `pizza` vs `Pizza`